upload/admin/view/template/extension/payment/emspay.tpl

```diff
@@ -281,6 +281,22 @@
                         </div>
                     </div>
 
+                    <?php if ($emspay_module == 'emspay_afterpay'): ?>
```

Comment:
This file already has `if block` for adding a custom testing data for Afterpay payment method, use that block, don't create a new block like this.

Reply:
Removed.